kolibri/core/content/management/commands/deletechannel.py

```diff
@@ -14,9 +15,46 @@
 class Command(AsyncCommand):
     def add_arguments(self, parser):
         parser.add_argument("channel_id", type=str)
+        # However, some optional arguments apply to both groups. Add them here!
+        node_ids_help_text = """
+        Specify one or more node IDs to delete. Only these ContentNodes and descendants will be deleted.
```

Comment:
perhaps rename the command to `deletecontent`, and add an alias to `deletechannel` for backward compatibility?

Reply:
Yeah, I had a similar thought. Will update.

Reply:
> I had a similar thought

Indeed you did :)

![image](https://user-images.githubusercontent.com/2367265/68410634-e7b12300-013d-11ea-9369-e13285bff62e.png)